[2024-08-20T10:39:15.481+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-20T10:39:15.536+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python.create_bucket manual__2024-08-20T10:39:14.127392+00:00 [queued]>
[2024-08-20T10:39:15.549+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python.create_bucket manual__2024-08-20T10:39:14.127392+00:00 [queued]>
[2024-08-20T10:39:15.550+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-20T10:39:15.576+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): create_bucket> on 2024-08-20 10:39:14.127392+00:00
[2024-08-20T10:39:15.590+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1597) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-20T10:39:15.592+0000] {standard_task_runner.py:72} INFO - Started process 1603 to run task
[2024-08-20T10:39:15.591+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python', 'create_bucket', 'manual__2024-08-20T10:39:14.127392+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_pyton.py', '--cfg-path', '/tmp/tmp_0zg94ap']
[2024-08-20T10:39:15.595+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask create_bucket
[2024-08-20T10:39:15.666+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_python.create_bucket manual__2024-08-20T10:39:14.127392+00:00 [running]> on host 93607a8c7584
[2024-08-20T10:39:15.828+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Juancito2085' AIRFLOW_CTX_DAG_ID='dag_with_python' AIRFLOW_CTX_TASK_ID='create_bucket' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T10:39:14.127392+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-20T10:39:14.127392+00:00'
[2024-08-20T10:39:15.829+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-20T10:39:15.854+0000] {logging_mixin.py:190} INFO - Bucket already exist at E:\Data Science\Tutorial-Apache Airflow\descargas
[2024-08-20T10:39:15.854+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-20T10:39:15.866+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-20T10:39:15.867+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_python, task_id=create_bucket, run_id=manual__2024-08-20T10:39:14.127392+00:00, execution_date=20240820T103914, start_date=20240820T103915, end_date=20240820T103915
[2024-08-20T10:39:15.930+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-20T10:39:15.949+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-20T10:39:15.951+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
